komga/src/main/kotlin/org/gotson/komga/domain/model/ThumbnailSeries.kt

```diff
@@ -6,12 +6,46 @@ import java.net.URL
 import java.time.LocalDateTime
 
 data class ThumbnailSeries(
-  val url: URL,
+  val url: URL?,
   val selected: Boolean = false,
+  var thumbnail: ByteArray?,
```

Comment:
Probably also need a type, like we have on the thumbnail book, to differentiate between sidecar and user uploaded.